zappa/core.py

```diff
@@ -754,11 +754,11 @@ def splitpath(path):
         archivef.close()
 
         # Trash the temp directory
-        shutil.rmtree(temp_project_path)
-        shutil.rmtree(temp_package_path)
+        shutil.rmtree(temp_project_path, ignore_errors=True)
+        shutil.rmtree(temp_package_path, ignore_errors=True)
         if os.path.isdir(venv) and slim_handler:
             # Remove the temporary handler venv folder
-            shutil.rmtree(venv)
+            shutil.rmtree(venv, ignore_errors=True)
```

Comment:
Can you share what kind of error these one causes? I wouldn't expect this to be a problem... kind of surprised.

Reply:
sometimes the folder either did not exist or is not empty, thatswhy ignore error to force deletion